frontend/src/components/main-page/groups-tree-page/common/dependencies/visualization/sankey-links.ts

```diff
@@ -62,15 +62,15 @@ export function buildRawLinks(
       The "natural" flow of Events if from producer to consumer:
       (SomeProducer) --> (Event) --> (SomeConsumer)
```

Comment:
```suggestion
      The "natural" flow of Events if from provider to subscriber:
      (SomeProvider) --> (Event) --> (SomeSubscriber)
```

Reply:
Addressed in [380ba05](https://github.com/osrgroup/msadoc/pull/75/commits/380ba05360e0fcb697ce53874c65ca76dfba6f0e)